fastlane/jetpack_metadata/fr-FR/subtitle.txt

```diff
@@ -0,0 +1 @@
+Site WP plus sûr, plus rapide.
```

Comment:
Totally opinionated, but this translation seems a bit rushed / too casual / familiar.

* The original is: `Make your WordPress site faster and safer.`
* This French translation could roughly be translated back to English as: `WP site safer, faster.`

Note the WP abbreviation, the inversion of faster and safer adjective order, the lack of verb… quite familiar talk.

Worth pining #i18n to make it fuzzy and ask for a more "professional"-sounding translation?
My personal take: `Rendez votre site WordPress plus rapide et plus sûr.`

Reply:
@AliSoftware I think you should make the suggestion and ping #i18n so they can take a look and approve. I believe all a12s are encouraged to do this.

Reply:
Thanks for the comment @AliSoftware and @oguzkocer 

FYI: the subtitle metadata was not submitted to App Store Connect because most of the localization (including the original English) were over the 30 characters limit. This has been reported in the Jetpack P2.

Reply:
Ah which means my suggestion above would be too long too 😓